discrete_summary_checker/example/Check_Format.ipynb

```diff
@@ -0,0 +1,956 @@
+       "3  *0000000000000100                                3425  NBP1709_001.hex   \n",
+       "4  *0000000000000100                                3425  NBP1709_001.hex   \n",
+       "\n",
+       "   ... Discrete pH Replicate Flag  Calculated Alkalinity [umol/kg]  \\\n",
+       "0  ...                   -9999999                         -9999999   \n",
+       "1  ...                   -9999999                         -9999999   \n",
+       "2  ...                   -9999999                         -9999999   \n",
+       "3  ...                   -9999999                         -9999999   \n",
+       "4  ...                   -9999999                         -9999999   \n",
+       "\n",
+       "  Calculated DIC [umol/kg] Calculated pCO2 [uatm]  Calculated pH  \\\n",
+       "0                 -9999999               -9999999       -9999999   \n",
+       "1                 -9999999               -9999999       -9999999   \n",
+       "2                 -9999999               -9999999       -9999999   \n",
+       "3                 -9999999               -9999999       -9999999   \n",
+       "4                 -9999999               -9999999       -9999999   \n",
+       "\n",
+       "  Calculated CO2aq [umol/kg]  Calculated Bicarb [umol/kg]  \\\n",
+       "0                   -9999999                     -9999999   \n",
+       "1                   -9999999                     -9999999   \n",
+       "2                   -9999999                     -9999999   \n",
+       "3                   -9999999                     -9999999   \n",
+       "4                   -9999999                     -9999999   \n",
+       "\n",
+       "   Calculated CO3 [umol/kg]  Calculated Omega-C  Calculated Omega-A  \n",
+       "0                  -9999999            -9999999            -9999999  \n",
+       "1                  -9999999            -9999999            -9999999  \n",
+       "2                  -9999999            -9999999            -9999999  \n",
+       "3                  -9999999            -9999999            -9999999  \n",
+       "4                  -9999999            -9999999            -9999999  \n",
+       "\n",
+       "[5 rows x 80 columns]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "summary_sheet = pd.read_excel(\"../data/Southern_Ocean-04_NBP1709_Discrete_Sample_Summary_2021-05-05_ACR.xlsx\")\n",
+    "summary_sheet.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Cruise Names\n",
+    "Load the R2R list of cruise names. These are the \"official\" cruise names which should be entered on the spreadsheets."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['UNRECOVERED',\n",
+       " 'TN299',\n",
```

Comment:
These are hard-coded in?  Not read in somehow from CruiseInformation.csv?

Reply:
I forgot that the .gitignore wasn't set to my standard, so it accidentally read in output cells from the notebook. It is not hardcoded.